src/UrqlTypes.re

```diff
@@ -41,10 +41,10 @@ type hookResponse('ret) = {
 };
 
 [@bs.deriving abstract]
-type jsResponse = {
+type jsResponse('response) = {
```

Comment:
Passing a type argument here allows us to use this type across all hooks – previously `useSubscription` couldn't use this type because the JS response from calling the `useSubscription` hook could be either a `Js.Json.t` (if no handler was provided), or the return type of the handler function (if a handler was provided). This just makes it more flexible.